bytecomp/lambda.ml

```diff
@@ -140,6 +140,10 @@ type primitive =
   | Pint_as_pointer
   (* Inhibition of optimisation *)
   | Popaque
+  (* Pointer operations: (assumes alignment) *)
+  | Pload8
+  | Pload16 of bool
```

Comment:
Could you add a type `alignment = Aligned | Unaligned`? I think we have too many booleans already